docs/configuration/extract-codacy-logs.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
+
+#### Helper functions ####
+
+usage()
+{
+    echo "Usage: $0 -n <NAMESPACE> [-d <DAYS_OF_LOGS>]"
+    echo "When using the -d flag, <DAYS_OF_LOGS> must be between 1 and 7"
```

Comment:
Why do we limit between 1 and 7 days?

We let customers configure the retention for this: `fluentdoperator.expirationDays` [here](https://github.com/codacy/chart) -> Also, should we explain in this doc how to configure this value also?

Reply:
I don't think that over 7 days of logs are valuable when debugging issues. I do understand that we might be sending mixed messages but I think that in the script we shouldn't allow to extract more than 7 days, since the way we pattern match to find the correct days to extract is not cheap.

Reply:
hmm.. then we should say that at least in README.md on the home page.
Retention over 7 days is not recommended and

Reply:
Ok, I'll add it there.